### DISCUSSION TITLE
the bot leaves the channel after playing music

### DISCUSSION THREAD

**Discussion by KauaAniceto** [jagrosh/MusicBot#791] [Questions] [answered]
hi, i would like to make the bot stay in the voice channel for a certain time after playing all the songs, how can i do that?

**Comment by MichailiK** [comment]
This is currently not possible.

**Comment by jagrosh** [comment, marked as answer]
You can enable the following settings in your config to have the bot stay in the channel until 60 seconds has elapsed that it is alone:
```
stayinchannel = true
alonetimeuntilstop = 60
```
(note that this isn't _exactly_ what you asked for; if people are still in the channel then bot won't leave; it will wait until it is alone or nobody is listening)
